src/main/java/core/Game.java

```diff
@@ -936,8 +952,19 @@ public static void main(String[] args) {
         players.add(new RandomPlayer());
         players.add(new RandomPlayer());
 //        players.add(new MCTSPlayer());
-//        MCTSParams params1 = new MCTSParams();
+        MCTSParams params1 = new MCTSParams();
+        params1.gatherTreeRecorder = true;
+        params1.discardStateAfterEachIteration = false;
```

Comment:
All fine - but advance warning of an incoming refactor that removed all the EIDG code from MCTS directly, and moves it to a standalone MCTSListener that ties in with the Metrics system